src/app/(full-page)/movies/[movieId]/page.tsx

```diff
@@ -1,125 +1,75 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
+import { GetServerSideProps } from "next";
+import { Movie } from "../../../../lib/types/movie";
+import { MovieCard } from "../../../../components/movie-card";
+import { PageButton } from "../../../../components/page-button";
 import axios from "axios";
-import Link from "next/link";
-import { MovieCredit, MovieDetail } from "@/lib/types/movie";
-import { BookmarkButton } from "./bookmark-button";
-import ReadMore from "@/components/read-more";
-import { protectPage } from "@/lib/auth";
 
-export default async function MovieDetails({
-  params,
-}: {
-  params: { movieId: string };
-}) {
-  await protectPage();
-  const { data: movie } = await axios.get<MovieDetail>(
-    `https://api.themoviedb.org/3/movie/${params.movieId}`,
-    {
-      params: {
-        api_key: process.env.TMDB_API_KEY,
-        language: "en-US",
-      },
-    }
-  );
+interface MoviesProps {
+  movies: Movie[];
+}
 
-  const credits = await axios
-    .get<MovieCredit>(
-      `https://api.themoviedb.org/3/movie/${params.movieId}/credits`,
-      {
-        params: {
-          api_key: process.env.TMDB_API_KEY,
-          language: "en-US",
-        },
-      }
-    )
-    .then((res) => res.data);
+const Movies = ({ movies }: MoviesProps) => {
```

Comment:
Where are you passing the movies from as a prop?
Ususally you cannot pass ciustom props to pages.
Also this is the incorrect page, this page is the movie detail page

Reply:
sry, uploaded the wrong code, was switching between projects and messed up :D